Warhammer-Fantasy-Roleplay-2nd-Edition/WHFRP2.html

```diff
@@ -268,28 +268,28 @@ <h4 class="sheet-center">
         <div class="sheet-col-1"><br/></div>
         
         <div class="sheet-col-11 sheet-pad-t-xsm sheet-pad-r-sm"></div>
-        <button type="roll" class="sheet-roll" name="roll_Weapon_Skill" value="&{template:whfrp2e} {{title=Weapon Skill}} {{character_name=@{character_name}}} {{target=[[@{WeaponSkill}+?{Modifier|0}]]}} {{chartest=[[1d100]]}} {{degree={target-]]}}">
+        <button type="roll" class="sheet-roll" name="roll_Weapon_Skill" value="&{template:whfrp2e} {{title=^{weapon-skill}}} {{character_name=@{character_name}}} {{target=[[@{WeaponSkill}+?{Modifier|0}]]}} {{chartest=[[1d100]]}} {{degree={target-]]}}">
           <span data-i18n="abbreviate-weapon-skill">WS</span>
         </button>
-        <button type="roll" class="sheet-roll" name="roll_Ballistic_Skill" value="&{template:whfrp2e} {{title=Ballistic Skill}} {{character_name=@{character_name}}} {{target=[[@{BallisticSkill}+?{Modifier|0}]]}} {{chartest=[[1d100]]}}">
+        <button type="roll" class="sheet-roll" name="roll_Ballistic_Skill" value="&{template:whfrp2e} {{title=^{ballistic-skill}}} {{character_name=@{character_name}}} {{target=[[@{BallisticSkill}+?{Modifier|0}]]}} {{chartest=[[1d100]]}}">
           <span data-i18n="abbreviate-ballistic-skill">BS</span>
         </button>
-        <button type="roll" class="sheet-roll" name="roll_Strength" value="&{template:whfrp2e} {{title=Strength}} {{character_name=@{character_name}}} {{target=[[@{Strength}+?{Modifier|0}]]}} {{chartest=[[1d100]]}}">
+        <button type="roll" class="sheet-roll" name="roll_Strength" value="&{template:whfrp2e} {{title=^{strength}}} {{character_name=@{character_name}}} {{target=[[@{Strength}+?{Modifier|0}]]}} {{chartest=[[1d100]]}}">
           <span data-i18n="abbreviate-strength">S</span>
         </button>&nbsp;&nbsp;
-        <button type="roll" class="sheet-roll" name="roll_Toughness" value="&{template:whfrp2e} {{title=Toughness}} {{character_name=@{character_name}}} {{target=[[@{Toughness}+?{Modifier|0}]]}} {{chartest=[[1d100]]}}">
+        <button type="roll" class="sheet-roll" name="roll_Toughness" value="&{template:whfrp2e} {{title=^{toughness}}} {{character_name=@{character_name}}} {{target=[[@{Toughness}+?{Modifier|0}]]}} {{chartest=[[1d100]]}}">
           <span data-i18n="abbreviate-toughness">T</span>
         </button>&nbsp;
-        <button type="roll" class="sheet-roll" name="roll_Agility" value="&{template:whfrp2e} {{title=Agility}} {{character_name=@{character_name}}} {{target=[[@{Agility}-@{armortestpenalty}+?{Modifier|0}]]}} {{chartest=[[1d100]]}}">
+        <button type="roll" class="sheet-roll" name="roll_Agility" value="&{template:whfrp2e} {{title=^{agility}}} {{character_name=@{character_name}}} {{target=[[@{Agility}-@{armortestpenalty}+?{Modifier|0}]]}} {{chartest=[[1d100]]}}">
```

Comment:
I didn't succeed to translate the question ask at roll.
`?{^{modifier}|0}` seems to not working.